blockchain/state_migration.go

```diff
@@ -122,14 +123,17 @@ func (bc *BlockChain) migrateState(rootHash common.Hash) error {
 		go bc.concurrentRead(srcCachedDB, quitCh, hashCh, resultCh)
 	}
 
+	stateTrieBatch := targetDB.DiskDB().NewBatch(database.StateTrieDB)
+	stats := migrationStats{initialStartTime: start, startTime: mclock.Now()}
+
 	// Migration main loop
 	for trieSync.Pending() > 0 {
 		bc.committedCnt, bc.pendingCnt = committedCnt, trieSync.Pending()
-		queue = append(queue[:0], trieSync.Missing(database.IdealBatchSize)...)
+		queue = append(queue[:0], trieSync.Missing(1024)...)
```

Comment:
We need to change CalcProgressPercentage() if the size of trieSync.Missing is changed.
I think the accuracy of depth 3 is enough.

Reply:
Good. I reflected.